src/server/service/diagnostics/sys.rs

```diff
@@ -479,6 +480,21 @@ fn get_sysctl_list() -> HashMap<String, String> {
         .collect()
 }
 
+fn get_transparent_hugepage() -> ServerInfoItem {
```

Comment:
This function should return `Option<ServerInfoItem>`.

Reply:
Agree

Reply:
addressed, PTAL